cfgov/paying_for_college/models/disclosures.py

```diff
@@ -15,13 +14,8 @@
 
 import requests
 
-
-if six.PY2:  # pragma: no cover
-    from unicodecsv import writer as csw  # pragma: no cover
-    from unicodecsv import DictReader as cdr  # noqa # pragma: no cover
-else:  # pragma: no cover
-    from csv import writer as csw  # pragma: no cover
-    from csv import DictReader as cdr  # noqa # pragma: no cover
+from csv import writer as csw  # pragma: no cover
```

Comment:
```suggestion
from csv import writer as csw
```

Reply:
No longer need to wave off coverage here (and in the next line as well)

Reply:
Done